namer/namer.cc

```diff
@@ -1432,6 +1483,14 @@ class SymbolDefiner {
         // all the `nonDeletableDefinitions` from all files get defined, then delete all the old
         // methods, then define all the methods.
         if (oldFoundHashes.has_value()) {
+            for (const auto &oldFieldHash : oldFoundHashes.value().fieldHashes) {
+                if (oldFieldHash.owner.isInstanceVariable) {
+                    deleteViaFullNameHash(ctx, oldFieldHash);
+                }
+            }
+
+            // TODO(froydnj): what happens if methods are deleted in one file but reappear
+            // in another?  It seems like this should be OK, but need to verify.
```

Comment:
Is this the bug we discussed on slack a few days ago, relating to the comment just above about not interleaving by file? Should we commit that test and/or update this comment to say that there is a known bug?

Reply:
I am happy to push a PR for that test, but wasn't sure how to handle the bit of it that fails, but shouldn't.  I guess commit as-is, but file an issue saying we have this expected error that actually shouldn't be there?

Reply:
The test is #6348.  I will remove this comment.